R-package/R/callback.R

```diff
@@ -173,6 +173,7 @@ cb.print.evaluation <- function(period = 1L) {
 
         # Check if message is existing
         if (nchar(msg) > 0L) {
+          print(merge.eval.string(env = env))
           cat(merge.eval.string(env = env), "\n")
```

Comment:
`cat` should be removed?

Reply:
:scream: how did that sneak in there! Probably a bad merge conflict resolution by me, thanks for catching it.

Reply:
fixed in https://github.com/microsoft/LightGBM/pull/3338/commits/9d9165aaea55bfcc33811502b2a4ff02c4b59fd1